src/views/Header.js

```diff
@@ -212,16 +220,34 @@ class Header extends React.Component<void, HeaderProps, void> {
       subView = defaultRenderer(subViewProps);
     }
 
-    if (subView === null) {
-      return null;
-    }
-
     const pointerEvents = offset !== 0 || isStale ? 'none' : 'box-none';
+
+    // On iOS, width of left/right components depends on the calculated
+    // size of the title.
+    const onLayoutIOS = Platform.OS === 'ios' && name === 'title'
```

Comment:
Define this function as a class property instead of inside render to avoid creating a new function every render